New documentation for 8 (#1215)
elastic/elasticsearch-php: docs/breaking-changes.asciidoc

[[breaking_changes]]
=== Breaking changes from 7.x

This major release is a complete new PHP client for {es}. We build it from scratch!
We tried to reduce the BC breaks as much as possible but there are some (big) differences: 

[discrete]
==== Architectural changes:

- we changed the namespace, now everything is under `Elastic\Elasticsearch`

> - we used the [elastic-transport-php](https://github.com/elastic/elastic-transport-php) library for HTTP communications;

Reviewer: MD --> Asciidoc

```suggestion
- we used the https://github.com/elastic/elastic-transport-php[elastic-transport-php] library for HTTP communications;
```

- we changed the `Exception` model, using the namespace `Elastic\Elasticsearch\Exception`. All the exceptions extends the
  `ElasticsearchException` interface, as in 7.x

> - we changed the response type of each endpoints using an [Elasticsearch](src/Response/Elasticsearch.php) response class.

Reviewer: I changed the link to an absolute one, please check if I used the proper one. I think it's from the readme file but it won't work on the rendered HTML page as a relative link. Also MD --> Asciidoc.

```suggestion
- we changed the response type of each endpoint using an https://github.com/elastic/elasticsearch-php/blob/master/src/Response/Elasticsearch.php[Elasticsearch] response class.
```

>   This class wraps a a [PSR-7](https://www.php-fig.org/psr/psr-7/) response allowing the access of the body response

Reviewer: MD --> Asciidoc

```suggestion
  This class wraps a https://www.php-fig.org/psr/psr-7/[PSR-7] response allowing the access of the body response
```
